lexisnexis-api-client.gemspec

```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |s|
     'Jonathan Hooper <jonathan.hooper@gsa.gov>',
   ]
   s.email = 'hello@login.gov'
-  s.homepage = 'http://github.com/18F/identity-aamva-api-client-gem'
```

Comment:
😂  good catch!

Reply:
Yeah...I was kinda like: "Hmmmm, I wonder what this file was copied from!"